src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReferenceShard.java

```diff
@@ -0,0 +1,92 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import com.google.cloud.dataflow.sdk.coders.*;
+import com.google.cloud.dataflow.sdk.values.KV;
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.Serializable;
+
+/**
+ * ReferenceShard is section of the reference genome that's used for sharding work for pairing things with
+ * the reference. The primary use case is pairing reads with the reference.
+ * This class designed to be simple a data-storage class with related static utilities.
+ */
+public class ReferenceShard implements Serializable {
```

Comment:
That comment sucks. I meant tweaked over time by humans, not dynamically. I'm fine removing static and/or making it final. That is, if we keep these shards instead of using `SimpleInterval`. I think we should talk with Louis and David over chat about this.